src/cli.yaml

```diff
@@ -206,7 +207,7 @@ subcommands:
       subcommands:
         - fastq:
             about: |
-              Tool to remove PCR duplicates from UMI-tagged reads.
+              Tool to predict maximum likelihood fragment sequence from UMI-tagged reads.
```

Comment:
Maybe extend to (feel free to correct the wording, if any of my assumptions are wrong!):

    Tool to merge sets of reads in paired FASTQ files that share the UMI and have similar read sequence. The result is a maximum likelihood fragment sequence per set.